apps/web/pages/booking/[uid].tsx

```diff
@@ -705,6 +710,24 @@ export default function Success(props: SuccessProps) {
                   </>
                 )}
               </div>
+              {isGmail && (
+                <Alert
+                  className="main -mb-20 mt-4 inline-block text-left sm:-mt-4 sm:mb-4 sm:w-full sm:max-w-xl sm:align-middle"
+                  severity="warning"
+                  message={
+                    <div>
+                      <p className="font-semibold">{t("google_new_spam_policy")}</p>
+                      <span className="underline">
+                        <Link href="https://cal.com/blog/google-s-new-spam-policy-may-be-affecting-your-invitations">
+                          {t("resolve")}
+                        </Link>
```

Comment:
```suggestion
                        <a target="_blank" href="https://cal.com/blog/google-s-new-spam-policy-may-be-affecting-your-invitations">
                          {t("resolve")}
                        </a>
```
Since it's a link outside of the web app we can use <a/> instead of <Link />